addons/mail/models/discuss/ir_binary.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models
+
+
+class IrBinary(models.AbstractModel):
+    _inherit = "ir.binary"
+
+    def _find_record_check_access(self, record, access_token, field):
+        if record._name in ["res.partner", "mail.guest"] and field == "avatar_128":
+            current_partner, current_guest = self.env["res.partner"]._get_current_persona()
+            record_rule = []
```

Comment:
I would just name this domain, record rule is confusing as it is another concept.

Reply:
I second this